tests/dummy/app/app.js

```diff
@@ -15,7 +15,7 @@ App = Ember.Application.extend({
 });
 
 // Legacy view-as-component weirdness. Don't ever ever ever do this.
-Ember.Handlebars.helper('carousel-item', CarouselItem);
+Ember.Helper.helper('carousel-item', CarouselItem);
```

Comment:
🤔 this works?! `Ember.Helper.helper` returns a helper function, usually in a file in `app/helpers/` and it can then be invoked in a template.

I don't think this is doing anything. Or if it is, wow.

Downstream we always use `{{#view 'carousel-item'}}` since `carousel-item` is in the views folder. Can the invocations of `{{carousel-item}}` in the dummy test app just be converted to that usage? Then this can certainly be deleted.

Reply:
I need to check the dummy app/help site.

It may work because the template and view doesn't do much. I'm also not sure that this is actually tested.

Reply:
Fixed